app/views/controllers/projects/aliases/show.html.erb

```diff
@@ -16,7 +16,7 @@
 
 <p>
   <strong>Target:</strong>
```

Comment:
If you want to use our stylesheets instead of html tag:
```suggestion
  <span class="font-weight-bold">Target:</span>
```

Reply:
Done.  Hadn't updated that template.  It's not terribly relevant any more, but I updated it just the same.  I did notice that we have a bunch of other `<strong>`s in the code base, but that's clearly outside the scope of this PR.